css/index.css

```diff
@@ -1,3 +1,290 @@
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* -------------------------------------------------------- */
+.container {
+  max-width: 880px;
+  width: 100%;
+  margin: 0 auto;
+}
+
+@media(max-width:800px){
+    .container{}
+}
+
+@media(max-width:500px){
+    .container{}
+}
+/* header */
+header{
+    display:flex;
+    width:100%;
+    justify-content: space-around;
+}
+
+header nav {
+  display: flex;
+  width: 70%;
+  justify-content: space-between;
+  margin-top: 35px;
+}
+
+header nav a {
+  /* margin-right: 65px; */
+  font-size: 14px;
+  color: gray;
+  text-decoration: none;
+}
+
+header nav a:hover {
+  text-decoration: underline;
+}
+
+header nav a:first-child {
+  margin-left: 0;
+}
+
+header nav a:last-child {
+  margin-right: 0;
+}
+
+header .logo {
+  display: flex;
+  margin-top: 25px;
+}
+
+@media(max-width:800px){
+    header{
+        flex-direction: column-reverse;
+        align-items: center;
+
+    }
+}
+
+@media(max-width:500px){
+    header{
+        justify-content: space-evenly;
+        margin:0;
+    }
+    header nav{
+        flex-direction: column;
+        width: 100%;
+        justify-content: space-evenly;
+        /* align-items: center; */
+        text-align: center;
+        padding:0 0;
+        margin: 0 0;
+    }
+    header nav a{
+        border: 1px solid #b0b0b0;
+        width: 100%;
+        padding: 3%;
+    }
+
+    header img{
+        margin: 2%;
+        padding: 0;
+    }
+}
+
+/* cta */
+.cta{
+    display: flex;
+    width:100%;
+    justify-content: space-around;
+}
+
+.cta .cta-text {
+  display: flex;
+  flex-direction: column;
+  font-size: 77px;
+  margin-top: 120px;
+  text-align: center;
+  padding: 0px 75px;
+}
+
+.cta .cta-text button {
+  background: white;
+  margin-top: 6px;
+  padding: 8px 44px;
+  font-size: 17px;
+  border: 1px solid black;
+  cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+  background: black;
+  color: white;
+}
+
+.cta img {
+  display:flex;
+  width: 40%;
+  margin-top: 69px;
```

Comment:
I played around with this in the dev tools because it is responsive and doesn't cause any scroll bars when shrinking the window, but having a circular image squish like it does now is a little strange behavior. you actually can remove all these styles except the `margin-top`, and you just need to remove the horizontal padding currently on `.cta .cta-text` and everything will respond as it should at all points